src/Utilities/CssConcatenator/MediaRule.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Utilities\CssConcatenator;
+
+/**
+ * Implementation of media rules for \Pelago\Emogrifier\Utilities\CssConcatenator
+ *
+ * @internal
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ * @author SignpostMarv
+ */
+class MediaRule
+{
+    /**
+     * The media query string, e.g. "@media screen and (max-width:639px)", or an empty string for
+     *   rules not within a media query block;
+     *
+     * @var string
+     */
+    public $media;
```

Comment:
possible `@todo` docblock tags:
* amend to `public string $media` when support for <= 7.3 is dropped
* review whether one should use `@readonly` when `vimeo/psalm` is upgraded as part of #793

Reply:
> possible `@todo` docblock tags

Some analysis tools may reject `@todo` in a DocBlock.  I'm pretty sure I've found some to reject `TODO` in a comment.

>     * amend to `public string $media` when support for <= 7.3 is dropped

There'll be lots of similar instances to update when that time comes.  Nothing more to do here.

>     * review whether one should use `@readonly` when `vimeo/psalm` is upgraded as part of #793

Possibly add a note there, but the fact you've x-referenced it should be sufficient :)